about.html

```diff
@@ -0,0 +1,107 @@
+          <ul class="burger-menu">
+            <li><a href="./index.html">Home</a></li>
+            <li><a href="#">News</a></li>
+            <li><a href="#">Videos</a></li>
+            <li><a href="#">Matches</a></li>
+            <li><a href="#">Teams</a></li>
+            <li><a href="#">Shop</a></li>
+          </ul>
+        </div>
+        <div class="navbars">
+          <ul class="nav1">
+            <li><a class="nav1-text" href="#">English</a></li>
+            <li><a class="nav1-text" href="#">Join Us</a></li>
+            <li><a class="nav1-text" href="#">Logout</a></li>
+          </ul>
+          <div class="nav2">
+            <a class="logo-home" href="./index.html">
+            <img src="./images/icons/logo.png" alt="logo"/></a>
+            <ul>
+              <li><a href="./index.html">Home</a></li>
+              <li><a href="#">News</a></li>
+              <li><a href="#">Videos</a></li>
+              <li><a href="#">Matches</a></li>
+              <li><a href="#">Teams</a></li>
+              <li><a href="#">Shop</a></li>
+            </ul>
+            <a class="zamalek-fc">Zamalek FC</a>
+          </div>
+        </div>
+        <div class="icon">
+          <div class="vector"></div>
+          <div class="vector"></div>
+          <div class="vector"></div>
+        </div>
+      </nav>
+    </header>
+    <main>
+      <section class="about center">
+        <div class="intro">
+          <p class="hello">"Hello! Zamalek fans"</p>
+          <h1>Election event for the most titled club in the 20th century.</h1>
+          <p class="header-description">
+            A general invitation to attend the Zamalek club elections for a new presidential term,
+            election festival , listen to candidates depates.
+          </p>
+
+          <p class="header-description center no-border">
+            Zamalek_fc_1911@gmail.com
+          </p>
+        </div>
+        <div class="about-logo">
+          <h2>Zamalek Election Logo</h2>
+          <span></span>
+          <p class="header-description center no-border">
+            The logo of Zamalek club 2022 was decided since 1911.
+          </p>
+          <img
+            class="event-logo"
+            src="./images/icons/logo.png"
+            alt="event logo"
+          />
+        </div>
+        <div class="about-logo">
+          <h2>See the past Elections</h2>
+          <span></span>
+          <p class="header-description center no-border">
+            Take a look at the last two election of zamalek club 2020 , 2021.
+          </p>
+          <div class="elections">
+            <div class="last-election1">
+              <p>2021</p>
+              <p>Zamalek election 2021</p>
+            </div>
+            <div class="last-election2">
+              <p>2020</p>
+              <p>Zamalek election 2020</p>
+            </div>
+          </div>
+        </div>
+      </section>
+    </main>
+    <footer class="hide-mobile">
```

Comment:
- [ ] Kindly add the footer as in the design for the mobile version and stick to the design on the desktop version. 
### Mobile version footer- Design
![image](https://user-images.githubusercontent.com/44057863/169667156-1b294c4c-c16d-4db3-821d-c4358df4a588.png)
### Desktop version footer - Design
![image](https://user-images.githubusercontent.com/44057863/169667146-05e51243-2cfa-4a28-ad30-8503dd9fe5ec.png)
### Mobile version - Impl
![image](https://user-images.githubusercontent.com/44057863/169667205-476632f4-ff95-4356-a8d9-1536ace64552.png)

### Desktop version - Impl
![image](https://user-images.githubusercontent.com/44057863/169667234-73a8161f-a10f-48df-b5f2-2eb13fb57901.png)